cime/src/drivers/mct/main/cime_comp_mod.F90

```diff
@@ -410,6 +411,8 @@ module cime_comp_mod
   logical  :: glc_c2_lnd             ! .true.  => glc to lnd coupling on
   logical  :: glc_c2_ocn             ! .true.  => glc to ocn coupling on
   logical  :: glc_c2_ice             ! .true.  => glc to ice coupling on
+  logical  :: glcshelf_c2_ocn        ! .true.  => glc ice shelf to ocn coupling on
```

Comment:
@JeremyFyke , what is the intended difference between glc_c2_ocn/ice and glcshelf_c2_ocn/ice?  I thought there was no existing GLC to OCN/ICE coupling.

Reply:
glc_c2_ocn and glc_c2_ice originally (and I think still) relate to iceberg calving routing.  So, I made a new coupler flag to differentiate and separate these couplings (with unique mapping files, etc.) from all the subshelf melting couplings (glcshelf_c2_ocn).